components/binary_sensor/ttp229_bsf.rst

```diff
@@ -0,0 +1,56 @@
+TTP229 Capacitive Touch Sensor
+==============================
+
+.. seo::
+    :description: Instructions for setting up TTP229 BSF Capacitive Touch Sensor
+    :image: ttp229_bsf.png
+    :keywords: TTP229_BSF
+
+The ``ttp229 BSF`` sensor platform allows you to use your TTP229 BSF `datasheet <https://www.sunrom.com/get/611100>`__
+Capacitive Touch Sensor with ESPHome.
```

Comment:
Please also add a link to the i2c ttp229 component here with `:doc:` (see other examples). And explain how the two are different.